website/docs/r/db_instance.html.markdown

```diff
@@ -27,6 +27,8 @@ See the AWS Docs on [RDS Instance Maintenance][instance-maintenance] for more in
 ~> **Note:** All arguments including the username and password will be stored in the raw state as plain-text.
 [Read more about sensitive data instate](https://www.terraform.io/docs/state/sensitive-data.html).
 
+-> **Note:** Write-Only attribute `password_wo` is available to use in place of `password`. Write-Only attributes are supported in HashiCorp Terraform 1.11.0 and later.
```

Comment:
Should we link to write only argument docs here? Maybe
https://developer.hashicorp.com/terraform/language/v1.11.x/resources/ephemeral#write-only-arguments